examples/plotting/file/airports_map.py

```diff
@@ -22,13 +17,14 @@
 tile_source = WMTSTileSource(**tile_options)
 
 # set to roughly extent of points
-x_range = Range1d(start=airports['x'].min() - 10000, end=airports['x'].max() + 10000)
-y_range = Range1d(start=airports['y'].min() - 10000, end=airports['y'].max() + 10000)
+x_range = Range1d(start=airports['x'].min() - 10000, end=airports['x'].max() + 10000, bounds=None)
+y_range = Range1d(start=airports['y'].min() - 10000, end=airports['y'].max() + 10000, bounds=None)
```

Comment:
Just to be clear, this is a workaround, not a fix? The plot should work with auto-bounds turned on? (Or alternatively: some plots should have a way to disable/express incompatibility with auto-bounds) I'd say a workaround is OK for the release, but let's make an issue to invetigate further  ping@birdsarah